examples/analytics/emr-eks-fsx-lustre/examples/spark-execute/fsx-static-spark.sh

```diff
@@ -0,0 +1,108 @@
+aws s3 sync ./spark-scripts/ "${SPARK_JOB_S3_PATH}/"
+
+#--------------------------------------------
+# Execute Spark job
+#--------------------------------------------
+
+if [[ $EMR_VIRTUAL_CLUSTER_ID != "" ]]; then
+  echo "Found Cluster $EMR_VIRTUAL_CLUSTER_NAME; Executing the Spark job now..."
+  aws emr-containers start-job-run \
+    --virtual-cluster-id $EMR_VIRTUAL_CLUSTER_ID \
+    --name $JOB_NAME \
+    --execution-role-arn $EMR_JOB_EXECUTION_ROLE_ARN \
+    --release-label $EMR_EKS_RELEASE_LABEL \
+    --job-driver '{
+      "sparkSubmitJobDriver": {
+        "entryPoint": "'"$SPARK_JOB_S3_PATH"'/scripts/sample-spark-taxi-trip.py",
+        "entryPointArguments": ["'"$SPARK_JOB_S3_PATH"'/input/taxi-trip-data/",
+          "'"$SPARK_JOB_S3_PATH"'/output/static/taxi-trip-data/"
+        ],
+        "sparkSubmitParameters": "--conf spark.executor.instances=2 --conf spark.executor.memory=2G --conf spark.executor.cores=1 --conf spark.driver.cores=1"
+      }
+   }' \
+    --configuration-overrides '{
+      "applicationConfiguration": [
+          {
+            "classification": "spark-defaults",
+            "properties": {
+              "spark.kubernetes.driver.podTemplateFile":"'"$SPARK_JOB_S3_PATH"'/pod-templates/fsx-static-spark-driver.yaml",
+              "spark.kubernetes.executor.podTemplateFile":"'"$SPARK_JOB_S3_PATH"'/pod-templates/fsx-static-spark-executor.yaml",
+              "spark.driver.memory":"2G",
+              "spark.kubernetes.executor.podNamePrefix":"ny-taxi-exec",
+              "spark.ui.prometheus.enabled":"true",
+              "spark.executor.processTreeMetrics.enabled":"true",
+              "spark.kubernetes.driver.annotation.prometheus.io/scrape":"true",
+              "spark.kubernetes.driver.annotation.prometheus.io/path":"/metrics/executors/prometheus/",
+              "spark.kubernetes.driver.annotation.prometheus.io/port":"4040",
+              "spark.kubernetes.driver.service.annotation.prometheus.io/scrape":"true",
+              "spark.kubernetes.driver.service.annotation.prometheus.io/path":"/metrics/driver/prometheus/",
+              "spark.kubernetes.driver.service.annotation.prometheus.io/port":"4040",
+              "spark.metrics.conf.*.sink.prometheusServlet.class":"org.apache.spark.metrics.sink.PrometheusServlet",
+              "spark.metrics.conf.*.sink.prometheusServlet.path":"/metrics/driver/prometheus/",
+              "spark.metrics.conf.master.sink.prometheusServlet.path":"/metrics/master/prometheus/",
+              "spark.metrics.conf.applications.sink.prometheusServlet.path":"/metrics/applications/prometheus/"
+            }
+          }
+        ],
+      "monitoringConfiguration": {
+        "persistentAppUI":"ENABLED",
+        "cloudWatchMonitoringConfiguration": {
+          "logGroupName":"'"$CW_LOG_GROUP"'",
+          "logStreamNamePrefix":"'"$JOB_NAME"'"
+        }
+      }
+    }'
+else
+  echo "Cluster is not in running state $EMR_VIRTUAL_CLUSTER_NAME"
+fi
```

Comment:
Yes, These CSV files are input for Spark jobs. `fsx-static-spark.sh`  will upload these small csv files to customer provided s3 bucket and process them as part of this example. We use this in the blog as well

Reply:
I don't think its a good idea to add in large CSV files in this repo. It can lead to long clone times and timeouts when resolving conflicts etc. Can we modify the steps to down the CSV file from somewhere, e.g. s3 bucket, etc.

Reply:
Yeah I  agree. We also cannot keep the data in public s3 buckets. I will try to find a way to download the test data from public data sets. Do you know any AWS public datasets location that wont change?

Reply:
@askulkarni2 Test data has been removed now. Shell script will download the data and sync with S3 for PySpark examples test